examples/tutorial_2_customproto.md

```diff
@@ -54,23 +60,23 @@ proc main() {.async, gcsafe.} =
     testProto = TestProto.new()
     switch1 = newStandardSwitch(rng=rng)
     switch2 = newStandardSwitch(rng=rng)
-  
+
   switch1.mount(testProto)
-  
+
   await switch1.start()
   await switch2.start()
-    
   let conn = await switch2.dial(switch1.peerInfo.peerId, switch1.peerInfo.addrs, TestCodec)
 
+  # conn is now a fully setup connection, we talk directly to the switch1 custom protocol handler
   await testProto.hello(conn)
 
-  # We must close the connection ourselves when we're done with it
+  # We must close the connections ourselves when we're done with it
   await conn.close()
 
-  await allFutures(switch1.stop(), switch2.stop()) # close connections and shutdown all transports
+  await allFutures(switch1.stop(), switch2.stop()) # shutdown all transports
```

Comment:
And libp2p will close all current connections during stop, so my comment was accurate :) though it probably need better explanation

Reply:
Yep, gotcha. I think I was assuming this comment was referring to the stream (`conn` created during `dial()`).